### ISSUE TITLE
Filter Condition Doesn't Support “=” ?

### ISSUE THREAD

**Issue by cococo2000** [vearch/vearch#799]
The filter condition in the [Vearch Python SDK](https://github.com/vearch/vearch/blob/1591f7a47022268300c531dc39eb03b8a8afd705/sdk/python/vearch/filter.py#L6) seems to include "=" as a filter option. However, after reviewing the [document query code](https://github.com/vearch/vearch/blob/1591f7a47022268300c531dc39eb03b8a8afd705/internal/router/document/doc_query.go#L70), it appears that Vearch filter does not support "=" and "LIKE".

**Comment by zcdb** [comment]
Yes. It's on our todo list. Maybe a comment will make it clearer. I will add a comment.
